ui/src/app/tasks-jobs/tasks/launch/builder/builder.component.ts

```diff
@@ -1022,4 +1030,53 @@ export class BuilderComponent implements OnInit, OnDestroy {
       this.launch.emit({props: this.getProperties(), args: this.getArguments()});
     }
   }
+
+  hasMigrations(): boolean {
+    return this.migrations && this.migrations.migratedMatch && this.migrations.migratedMatch.length > 0;
+  }
+
+  migrage() {
```

Comment:
migrate?

Reply:
hah, funny how you don't see these typos.

Reply:
I hear ya'.   Done it more times that I can remember :-)